Moggles.Domain/Application.cs

```diff
@@ -17,7 +17,7 @@ public void UpdateName(string newName)
             AppName = newName;
         }
 
-        public void AddDeployEnvironment(string name, bool defaultToggleValue, int sortOrder = 1)
+        public void AddDeployEnvironment(string name, bool defaultToggleValue, string username, int sortOrder = 1)
```

Comment:
I think it is more obvious what this user name represents if you call the parameter name updatedBy